docs/maps/api-player.md

```diff
@@ -167,29 +167,160 @@ WA.player.onPlayerMove(console.log);
 ```
 
 ## Player specific variables
+
 Similarly to maps (see [API state related functions](api-state.md)), it is possible to store data **related to a specific player** in a "state". Such data will be stored using the local storage from the user's browser. Any value that is serializable in JSON can be stored.
 
+Any value that is serializable in JSON can be stored.
+
+Each variable can be stored and fetched in a variety of ways.
+
+Here is what defines a player variable.
+
+**Visibility:**
+
+A player variable can be **public** or **private**.
+- Public variables are automatically shared to players around you. Players around you can view
+these variables using the `RemotePlayer.state` object (you can get a `RemotePlayer` object) using
+[`WA.players.list()`](api-players.md).
+- Private variables are only accessible by the current user.
+
+**Persistence:**
+
+A player variable can be **persisted** or **transient**
+
+- Persisted variables are stored across sessions. If you refresh your page or come back later,
+  a persisted variable can be fetched again. Use "persisted variables" to store valuable values
+  (like a score in a game that is played in the long run)
+- Transient variables disappear as soon as the connection to the room is lost. So if you
+  you refresh your page, if your network connection is lost for a brief amount of time, or
+  if you simply close WorkAdventure and come back later, the transient player variable value
+  will be lost. Use transient variables for values that are short-lived in inherently tied to
+  the game state. For instance, if you are doing a live voting system, you can use a transient
+  variable to store the current vote of the player.
+
+**Time to live:**
+
+Persisted variables can have a **Time to live** (TTL):
+
+The TTL (expressed in seconds) is the time after which the stored value will be destroyed.
+TTL cannot be set on transient variables.
```

Comment:
They can be set on transient variables **only**?
If so add "only"
If not add "on persisted and transient variables." or remove the sentence.

Reply:
I replaced the sentence with:

TTL can be set on persisted variables only. It cannot be set on transient variables.